### DISCUSSION TITLE
Conditions in the playground

### DISCUSSION THREAD

**Discussion by zalsader** [openfga/community#404] [Ideas]
Hello!
I tried running the example in https://openfga.dev/docs/modeling/conditions in the playground, but there is no way to add context in tuples or checks. Is this planned?
Thank you

**Comment by rhamzeh** [comment]
Hi @zalsader - we're not planning on any near-term enhancements to the Playground.

We do have a plan to OSS the Playground, in which we'll probably rewrite it and add requested features such as context https://github.com/orgs/openfga/projects/1/views/1?filterQuery=play&pane=issue&itemId=5723543&issue=openfga%7Croadmap%7C7

However it's not high up on our priority list.

What I would recommend is using the OpenFGA CLI + VS Code/IntelliJ extension to write tests in the store format and run them using the CLI:
https://github.com/openfga/cli/?tab=readme-ov-file#run-tests-on-an-authorization-model

**Reply by zalsader** [reply]
thank you. I'll wait for the open source version and probably contribute that, thank you.
